source/README.adoc

```diff
@@ -0,0 +1,142 @@
+
+:toc:
+:toc-placement!:
+= RabbitMQ Source
+
+The RabbitMQ source translates messages on a RabbitMQ exchange to CloudEvents
+over HTTP for use with Knative Eventing. The Source can bind to an existing
+exchange or create a new one, as needed.
```

Comment:
```suggestion
The RabbitMQ source translates messages from a RabbitMQ exchange to CloudEvents, which can then be used with Knative Eventing over HTTP.
The source can bind to an existing RabbitMQ exchange, or create a new exchange if required.
```

Reply:
(This is suggesting sentence-per-line?)

Reply:
If you look at it it's actually suggestions for slightly different wording and to make it more consistent, but yeah I'd format this as suggested to make it more readable. Up to you though.